Setup GoMFT  [ERROR] in line 45: exit code 0:

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

GoMFT

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/gomft.sh)"

### 📝 Provide a clear and concise description of the issue.

I get this while executing the installation script

[ERROR] in line 45: exit code 0: while executing command wget -q "https://github.com/StarFleetCPTN/GoMFT/releases/download/v${RELEASE}/gomft-v${RELEASE}-linux-amd64" -O gomft

and when I try 
root@pve1:~# echo "$RELEASE"
I get nothing

Probably the RELEASE variable isn't getting parsed correctly

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

execute the installation script both default and advanced

### ❌ Paste the full error output (if available).

🧩  Using Advanced Settings on node pve1
  🖥️  Operating System: debian
  🌟  Version: 12
  📦  Container Type: Unprivileged
  🔐  Root Password: 
  🆔  Container ID: XXX
  🏠  Hostname: gomft
  💾  Disk Size: 4 GB
  🧠  CPU Cores: 1
  🛠️  RAM Size: 512 MiB
  🌉  Bridge: vmbr0
  📡  IP Address: dhcp
  🌐  Gateway IP Address: Default
  📡  APT-Cacher IP Address: Default
  🚫  Disable IPv6: no
  ⚙️  Interface MTU Size: Default
  🔍  DNS Search Domain: Host
  📡  DNS Server IP Address: Host
  🏷️  Vlan: Default
  📡  Tags: community-script;backup
  🔑  Root SSH Access: no
  🔍  Verbose Mode: no
  🚀  Creating a GoMFT LXC using the above advanced settings
  ✔️  Using local for Template Storage.
  ✔️  Using XXXX for Container Storage.
  ✔️  Updated LXC Template List
  ✔️  LXC Template is ready to use.
  ✔️  LXC Container 110 was successfully created.
  ✔️  Started LXC Container
  ✔️  Set up Container OS
  ✔️  Network Connected: XXX.XXX.XXX.XXX 
  ✔️  IPv4 Internet Connected
  ✖️  IPv6 Internet Not Connected
  ✔️  DNS Resolved github.com to XXX.XXX.XXX.XXX
  ✔️  Updated Container OS
  ✔️  Installed Dependencies
  ✔️  Setup Golang
 ⠏ Setup GoMFT 
[ERROR] in line 45: exit code 0: while executing command wget -q "https://github.com/StarFleetCPTN/GoMFT/releases/download/v${RELEASE}/gomft-v${RELEASE}-linux-amd64" -O gomft

### 🖼️ Additional context (optional).

_No response_

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -20,51 +20,51 @@ color
 catch_errors
 
 function update_script() {
-    header_info
-    check_container_storage
-    check_container_resources
+  header_info
+  check_container_storage
+  check_container_resources
 
-    if [[ ! -d "/opt/gomft" ]]; then
-        msg_error "No ${APP} Installation Found!"
-        exit
-    fi
-    RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
-    if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
-        msg_info "Stopping $APP"
-        systemctl stop gomft
-        msg_ok "Stopped $APP"
-
-        msg_info "Updating $APP to ${RELEASE}"
-        rm -f /opt/gomft/gomft
-        temp_file=$(mktemp)
-        wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
-        tar -xzf $temp_file
-        cp -rf GoMFT-${RELEASE}/* /opt/gomft
-        cd /opt/gomft
-        $STD go mod download
-        $STD go install github.com/a-h/templ/cmd/templ@latest
-        $STD $HOME/go/bin/templ generate
-        export CGO_ENABLED=1
-        export GOOS=linux
-        $STD go build -o gomft
-        chmod +x /opt/gomft/gomft
-        echo "${RELEASE}" >/opt/${APP}_version.txt
-        msg_ok "Updated $APP to ${RELEASE}"
-
-        msg_info "Cleaning Up"
-        rm -f $temp_file
-        rm -rf GoMFT-${RELEASE}
-        msg_ok "Cleanup Complete"
-
-        msg_info "Starting $APP"
-        systemctl start gomft
-        msg_ok "Started $APP"
-
-        msg_ok "Update Successful"
-    else
-        msg_ok "No update required. ${APP} is already at ${RELEASE}"
-    fi
+  if [[ ! -d "/opt/gomft" ]]; then
+    msg_error "No ${APP} Installation Found!"
     exit
+  fi
+  RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
+    msg_info "Stopping $APP"
+    systemctl stop gomft
+    msg_ok "Stopped $APP"
+
+    msg_info "Updating $APP to ${RELEASE}"
+    rm -f /opt/gomft/gomft
+    temp_file=$(mktemp)
+    wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
+    tar -xzf $temp_file
+    cp -rf GoMFT-${RELEASE}/* /opt/gomft
+    cd /opt/gomft
+    $STD go mod download
+    $STD go install github.com/a-h/templ/cmd/templ@latest
+    $STD $HOME/go/bin/templ generate
+    export CGO_ENABLED=1
+    export GOOS=linux
+    $STD go build -o gomft
+    chmod +x /opt/gomft/gomft
+    echo "${RELEASE}" >/opt/${APP}_version.txt
+    msg_ok "Updated $APP to ${RELEASE}"
+
+    msg_info "Cleaning Up"
+    rm -f $temp_file
+    rm -rf GoMFT-${RELEASE}
+    msg_ok "Cleanup Complete"
+
+    msg_info "Starting $APP"
+    systemctl start gomft
+    msg_ok "Started $APP"
+
+    msg_ok "Update Successful"
+  else
+    msg_ok "No update required. ${APP} is already at ${RELEASE}"
+  fi
+  exit
 }
 
 start

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -41,9 +41,17 @@ function update_script() {
         tar -xzf $temp_file
         cp -rf GoMFT-${RELEASE}/* /opt/gomft
         cd /opt/gomft
+<<<<<<< HEAD
         $STD go mod download
         $STD go build -o gomft
         $STD $HOME/go/bin/templ generate
+=======
+        $STD $HOME/go/bin/templ generate
+        export CGO_ENABLED=1
+        export GOOS=linux
+        $STD go mod download
+        $STD go build -o gomft
+>>>>>>> 67217457 (Fix dependency)
         chmod +x /opt/gomft/gomft
         echo "${RELEASE}" >/opt/${APP}_version.txt
         msg_ok "Updated $APP to ${RELEASE}"

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -41,17 +41,12 @@ function update_script() {
         tar -xzf $temp_file
         cp -rf GoMFT-${RELEASE}/* /opt/gomft
         cd /opt/gomft
-<<<<<<< HEAD
         $STD go mod download
-        $STD go build -o gomft
-        $STD $HOME/go/bin/templ generate
-=======
+        $STD go install github.com/a-h/templ/cmd/templ@latest
         $STD $HOME/go/bin/templ generate
         export CGO_ENABLED=1
         export GOOS=linux
-        $STD go mod download
         $STD go build -o gomft
->>>>>>> 67217457 (Fix dependency)
         chmod +x /opt/gomft/gomft
         echo "${RELEASE}" >/opt/${APP}_version.txt
         msg_ok "Updated $APP to ${RELEASE}"

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -15,14 +15,14 @@ update_os
 
 msg_info "Installing Dependencies"
 $STD apt-get install -y \
-    curl \
-    sudo \
-    mc \
-    sqlite3 \
-    rclone \
-    tzdata \
-    ca-certificates \
-    build-essential
+  curl \
+  sudo \
+  mc \
+  sqlite3 \
+  rclone \
+  tzdata \
+  ca-certificates \
+  build-essential
 msg_ok "Installed Dependencies"
 
 msg_info "Setting up Golang"

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -35,11 +35,7 @@ ln -sf /usr/local/go/bin/go /usr/local/bin/go
 set -o pipefail
 msg_ok "Setup Golang"
 
-<<<<<<< HEAD
-msg_info "Setup ${APPLICATION}"
-=======
 msg_info "Setup ${APPLICATION} (Patience)"
->>>>>>> 67217457 (Fix dependency)
 temp_file=$(mktemp)
 RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
 wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
@@ -47,18 +43,11 @@ tar -xzf $temp_file
 mv GoMFT-${RELEASE}/ /opt/gomft
 cd /opt/gomft
 $STD go mod download
-<<<<<<< HEAD
-$STD go build -o gomft
-chmod +x gomft
-$STD go install github.com/a-h/templ/cmd/templ@latest
-$STD $HOME/go/bin/templ generate
-=======
 $STD go install github.com/a-h/templ/cmd/templ@latest
 $STD $HOME/go/bin/templ generate
 export CGO_ENABLED=1
 export GOOS=linux
 $STD go build -o gomft
->>>>>>> 67217457 (Fix dependency)
 JWT_SECRET_KEY=$(openssl rand -base64 24 | tr -d '/+=')
 
 cat <<EOF >/opt/gomft/.env

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -48,6 +48,7 @@ $STD $HOME/go/bin/templ generate
 export CGO_ENABLED=1
 export GOOS=linux
 $STD go build -o gomft
+chmod +x /opt/gomft/gomft
 JWT_SECRET_KEY=$(openssl rand -base64 24 | tr -d '/+=')
 
 cat <<EOF >/opt/gomft/.env

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -21,7 +21,8 @@ $STD apt-get install -y \
     sqlite3 \
     rclone \
     tzdata \
-    ca-certificates
+    ca-certificates \
+    build-essential
 msg_ok "Installed Dependencies"
 
 msg_info "Setting up Golang"
@@ -34,7 +35,11 @@ ln -sf /usr/local/go/bin/go /usr/local/bin/go
 set -o pipefail
 msg_ok "Setup Golang"
 
+<<<<<<< HEAD
 msg_info "Setup ${APPLICATION}"
+=======
+msg_info "Setup ${APPLICATION} (Patience)"
+>>>>>>> 67217457 (Fix dependency)
 temp_file=$(mktemp)
 RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
 wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
@@ -42,10 +47,18 @@ tar -xzf $temp_file
 mv GoMFT-${RELEASE}/ /opt/gomft
 cd /opt/gomft
 $STD go mod download
+<<<<<<< HEAD
 $STD go build -o gomft
 chmod +x gomft
 $STD go install github.com/a-h/templ/cmd/templ@latest
 $STD $HOME/go/bin/templ generate
+=======
+$STD go install github.com/a-h/templ/cmd/templ@latest
+$STD $HOME/go/bin/templ generate
+export CGO_ENABLED=1
+export GOOS=linux
+$STD go build -o gomft
+>>>>>>> 67217457 (Fix dependency)
 JWT_SECRET_KEY=$(openssl rand -base64 24 | tr -d '/+=')
 
 cat <<EOF >/opt/gomft/.env

